From dc6f31f83df3a0ff2a9da93ffb61aee3e58ae852 Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Fri, 6 May 2022 10:30:09 +0800
Subject: [PATCH] sw64: kapi: remove redudant SMP_CACHE_BYTES
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: dc6f31f83df3a0ff2a9da93ffb61aee3e58ae852
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

SMP_CACHE_BYTES is also defined in linux/cache.h, so remove the
replicated one.

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/asm/cache.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/sw_64/include/asm/cache.h b/arch/sw_64/include/asm/cache.h
index a59a74110884..1dca2e2e04a4 100644
--- a/arch/sw_64/include/asm/cache.h
+++ b/arch/sw_64/include/asm/cache.h
@@ -5,9 +5,7 @@
 #ifndef _ASM_SW64_CACHE_H
 #define _ASM_SW64_CACHE_H
 
-#define L1_CACHE_BYTES		128
 #define L1_CACHE_SHIFT		7
-
-#define SMP_CACHE_BYTES		L1_CACHE_BYTES
+#define L1_CACHE_BYTES		(1 << L1_CACHE_SHIFT)
 
 #endif
-- 
2.33.0

